lms/djangoapps/course_wiki/tab.py

```diff
@@ -28,4 +28,10 @@ def is_enabled(cls, course, user=None):
             return False
         if course.allow_public_wiki_access:
             return True
-        return super(WikiCourseViewType, cls).is_enabled(course, user=user)
+        return super(WikiTab, cls).is_enabled(course, user=user)
+
+    def to_json(self):
```

Comment:
Can you move this up to ``CourseTab``? It seems like this will be necessary for any tab that supports ``is_hidden``.